modules/test/playwright/tests/object-web/objectFields.spec.ts

```diff
@@ -581,10 +587,51 @@ test.describe('Manage objectFields through Objects Admin UI', () => {
 			});
 		}
 
+		while (
+			(await page.locator('.dnd-tbody > .dnd-tr').all()).length !==
+			objectDefinition.objectFields.length + objectFieldsMock.length
+		) {
+			await page.waitForTimeout(1000);
+		}
```

Comment:
Hey @danvb112, I'm facing problems when I try to make this change here.

I'm testing locally using the show browser option. `objectFieldTableRows` has fewer expected items using just networkidle.

Reply:
Ok @igor-franca , so you can keep your original solution :smile: